saas/api/organizations.py

```diff
@@ -130,6 +132,12 @@ def put(self, request, *args, **kwargs):
               "full_name": "Xia Lee"
             }
         """
+        storage = get_picture_storage()
+        picture = request.data.get('picture')
```

Comment:
Why are we not using a serializer and `validated_data` here?

Reply:
Because letting it pass the data to subsequent methods would make the request invalid: serializer expects a url string not a file. We’ve experienced this issue while working on djaodjin-signup.

Reply:
There are no reasons why the serializer field couldn't have a different type as the model field.